### ISSUE TITLE
AppImage/Flatpak/Snap/anything please.

### ISSUE THREAD

**Issue by qwertea** [YACReader/yacreader#45]
It was quite a pain to install this from the AUR, took like an hour to compile. Alternatively, a precompiled binary package for Arch would be great as well.

**Comment by Ashutosh-Kukreti** [comment]
I think creating snap packages will be a nice idea since now they are available by default in the ubuntu software center,Although I like flatpak more because they are more secure.

**Comment by selmf** [comment]
I'm personally not a fan of these bundled "packages" (too much bloat for my taste), but the openSuse Build Service I use to build the YACReader packages does support AppImage targets, so I might look into it. It's not a priority though, so if anyone likes to give this a go - knock yourselves out ;)

**Comment by djkazaz** [comment]
I would definitely support it, especially flatpak, though appimages are ok too. 

Snaps do have a lot of bloat which hurts performance, something to avoid in an application like yacreader.

I've also had a lot of trouble getting it to work from the AUR and given the amount of fragmentation in Linux at the moment, I think universal packages are definitely the future.

**Comment by selmf** [comment]
Small update: We're currently re-evaluating this since we are about to take the leap towards Qt6 and this is not widely available on Linux yet. I'm still not a fan of universal packages, but it is a good way to not leave the users behind in this case.

**Comment by djkazaz** [comment]
I'm glad to hear it, because as I posted on the forum, I've tried everything to get yacreader to run on manjaro and failed.  

As for the question universal packages, I am not sure why people don't like them. Given the number of distributions currently floating around and the low cost of disk space, they make a lot of sense. Not everyone is on Ubuntu/debian, the fedora builds are out of date (they only go up to v31) and the AUR is obviously not the most reliable way to distribute an app.

A universal package solves this once and for all. Anyway, I will look forward to it. Thanks!
